app/models/ems_event.rb

```diff
@@ -104,7 +104,7 @@ def self.bottleneck_event_groups
   end
 
   def self.add_queue(meth, ems_id, event)
-    if MiqQueue.messaging_client('event_handler').present?
+    if MiqQueue.messaging_client('event_handler').present? && MiqQueueWorkerBase.worker_settings[:dequeue_method] != "drb"
```

Comment:
Since we care about the event handler specifically
```suggestion
    if MiqQueue.messaging_client('event_handler').present? && MiqEventHandler.worker_settings[:dequeue_method] != "drb"
```